bool isMoving
time now
ArmPose pose